src/libav.h

```diff
@@ -54,6 +54,13 @@ This list must be updated every time we use a new AV_CODEC_ID
 #define AV_CODEC_ID_DVB_TELETEXT CODEC_ID_DVB_TELETEXT
 #endif
 
+// Enable new transcoding starting ffmpeg 6
+#if LIBAVCODEC_VERSION_MAJOR < 60
+#define ENABLE_NEW_TRANSCODING 0
```

Comment:
Can we not use the `#if LIBAVCODEC_VERSION_MAJOR < 60` test everywhere instead of adding a new variable? I'm generally against calling things "new" in code because the new method is inevitably superseded and becomes another old method at some point requiring a future "new2" or "nextgen" naming silliness.

Reply:
@chewitt let me clarify why I choose this solution (maybe not the best): I don't know how far back this new methodology of transcoding works ... so in order to easily test and update later on, is better to update a single line that will allow users to test functionality. Maybe the name was not the best option ... frankly I am open to any naming convention. ENABLE_TRANSCODING_V2 ? this is a little mode future proof. 
More importantly: @Flole998 do you think the failures are real failures?